lib/python/alice_populate.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+from sled import Conf
+
+c = Conf()
+c.path(b"ALICE.data")
+t = c.tree()
+t.set(b"k1", b"v1")
+t.close()
```

Comment:
this little script just writes a single key to the file `ALICE.data` which the ALICE framework will later permute and feed to the checker for verification